api/client/system/info.go

```diff
@@ -86,6 +86,7 @@ func runInfo(dockerCli *client.DockerCli) error {
 			fmt.Fprintf(dockerCli.Out(), " Managers: %d\n", info.Swarm.Managers)
 			fmt.Fprintf(dockerCli.Out(), " Nodes: %d\n", info.Swarm.Nodes)
 		}
+		fmt.Fprintf(dockerCli.Out(), " Node Address: %s\n", info.Swarm.NodeAddr)
```

Comment:
@aaronlehmann shouldnt we expose both the LocalAddress (which is supposed to be the node address) and also the advertise-address ? I think that will be useful for cases such as using public-ip for advertise.